packages/d/docsearch-css.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "docsearch-css",
+  "filename": "style.css",
+  "description": "Styles for DocSearch.",
+  "keywords": [
+    "doc",
+    "search"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/algolia/docsearch.git"
+  },
+  "homepage": "https://docsearch.algolia.com/",
+  "autoupdate": {
+    "source": "npm",
+    "target": "@docsearch/css",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

Comment:
👋 We try to avoid using open globstars, as it can lead to including files we didn't intend to include on the CDN -- it looks like `*.css` would match everything needed here?

Reply:
> 👋 We try to avoid using open globstars, as it can lead to including files we didn't intend to include on the CDN -- it looks like `*.css` would match everything needed here?

yes, all css.

Reply:
Okay, if this is all CSS, please update the glob pattern to be `*.css` as I had suggested, we don't want open globstars as you currently have.

Reply:
> Okay, if this is all CSS, please update the glob pattern to be `*.css` as I had suggested, we don't want open globstars as you currently have.

ok, it done